containers/src/Kokkos_ScatterView.hpp

```diff
@@ -766,6 +778,14 @@ class ScatterView<DataType, Layout, DeviceType, Op, ScatterNonDuplicated,
   ScatterView(std::string const& name, Dims... dims)
       : internal_view(name, dims...) {}
 
+  template <typename... P, typename... Dims>
+  ScatterView(::Kokkos::Impl::ViewCtorProp<P...> const& arg_prop, Dims... dims)
+      : internal_view(arg_prop, dims...) {
+    using ::Kokkos::Impl::Experimental::
+        check_scatter_view_allocation_properties_argument;
+    check_scatter_view_allocation_properties_argument(arg_prop);
+  }
+
```

Comment:
Pass `Kokkos::view_alloc(exec_space, "label")`.  We need to add a test for it.

Reply:
Are you adding that test, or should I? I don't know how Kokkos goes about testing that various things happened on the right stream

Reply:
I wanted to discuss this point with the Kokkos team.  The ScatterView unit tests already take a ridiculous amount of time to run and I wanted to suggest we only add a compile-only check.  I think it is fine because we essentially defer the implementation to the overloads that take an execution space so all code path gets covered with the pre-existing unit tests.

Reply:
I agree.

Reply:
agreed